snforge_std/src/fs/file_operations.cairo

```diff
@@ -36,7 +33,7 @@ fn read_txt(file: @File) -> Array<felt252> {
 /// Returns an array of felts read from the file, panics if read was not possible, or json was incorrect
 fn read_json(file: @File) -> Array<felt252> {
```

Comment:
This one as well as the one above, preferably both wrapped with a trait

Reply:
@Arcticae This means I am taking out the `FileTrait` trait and bringing back the `File` trait right, and then move `read_txt` and `read_json` into the newly created `File` trait right?